binding.gyp

```diff
@@ -39,8 +39,10 @@
         }],
         ['OS=="mac" or OS=="solaris"', {
           'xcode_settings': {
-            'GCC_ENABLE_CPP_EXCEPTIONS': 'YES'
+            'GCC_ENABLE_CPP_EXCEPTIONS': 'YES',
+            'MACOSX_DEPLOYMENT_TARGET': '10.6',
```

Comment:
10.6 still seems to be the standard baseline for most things (it's still the target for Python.org Pythons, and thus all Python extensions built with them). For regular C/C++, this makes very little (if any) difference unless you are using Apple's Frameworks, which libzmq doesn't. I think the main effect is selecting the version of the compiler, and possibly whether it will load stdlib from `libc++` or `libstdc++`.

I think the convention is to keep using 10.6 as a magic number until it stops working, which it hasn't seemed to yet.